<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResearchGrantProposalsClass</class>
 <widget class="QMainWindow" name="ResearchGrantProposalsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ResearchGrantProposals</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>551</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="position_label">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLineEdit" name="duration_line_edit"/>
     </item>
     <item row="8" column="0">
      <widget class="QPushButton" name="develop_button">
       <property name="text">
        <string>Develop</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QPushButton" name="save_button">
       <property name="text">
        <string>Save all ideas</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="3">
      <widget class="QPushButton" name="add_idea_button">
       <property name="text">
        <string>add idea</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="decssription_line_edit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="title_label">
       <property name="text">
        <string>title</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="description_label">
       <property name="text">
        <string>description</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="duration_label">
       <property name="text">
        <string>duration</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLineEdit" name="title_lineEdit"/>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QTableView" name="ideas_tableView"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>894</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ResearchGrantProposals.qrc"/>
 </resources>
 <connections/>
</ui>
